terra-jupyter-python/Dockerfile

```diff
@@ -80,6 +80,9 @@ RUN pip3 -V \
  && pip3 install setuptools==42.0.2 \
  && pip3 install wheel
 
+ENV HTSLIB_CONFIGURE_OPTIONS="--enable-gcs"
+pip3 install Cython && pip3 install pysam==0.15.4 --no-binary pysam
```

Comment:
I think this needs to be prefixed with `RUN`?

Actually, can we put the `ENV` up above and chain this `pip3 install` in the existing layer? Something like:
```
ENV HTSLIB_CONFIGURE_OPTIONS="--enable-gcs"

RUN pip3 -V \
 && pip3 install --upgrade pip \
 … all the stuff …
 && pip3 install Cython \
 && pip3 install pysam==0.15.4 --no-binary
```

Reply:
Sounds good. Yes, sorry about the missing RUN. Apparently people tested a slightly different dockerfile without this mistake 😓

Reply:
OK, I think I addressed that now. Thanks!